packages/manager/modules/pci/src/projects/project/project.constants.js

```diff
@@ -83,7 +83,17 @@ export const LINKS = [
   },
 ];
 
+export const PCI_HDS_ADDON = {
+  productName: 'cloud',
+  family: 'certification-hds',
+  parentPlanCode: 'project.2018',
+  planCodeScope: 'certification.hds',
+  certifiedProject: 'publiccloud-certification-hds',
+  planCode: 'certification.hds.2018',
+};
```

Comment:
Shouldn't those info be fetched from API ?

Reply:
No, i need them to get info from API and compare info.